### PR TITLE
Export render stats: percentDone, numCompletedSamples

### PR DESCRIPTION
**Pull request by hshakula**
Related to #101

### CODE REVIEW

File: pxr/imaging/plugin/hdRpr/renderDelegate.cpp [hunk 1]
```diff
@@ -306,6 +307,14 @@ HdRenderSettingDescriptorList HdRprDelegate::GetRenderSettingDescriptors() const
     return m_settingDescriptors;
 }
 
+VtDictionary HdRprDelegate::GetRenderStats() const {
+    VtDictionary stats;
+    int numCompletedSamples = m_rprApiSharedPtr->GetNumCompletedSamples();
+    stats[HdPerfTokens->numCompletedSamples.GetString()] = numCompletedSamples;
+    stats[_tokens->percentDone.GetString()] = 100.0 * double(numCompletedSamples) / HdRprConfig::GetInstance().GetMaxSamples();
```
Comment (bsavery) [inline pxr/imaging/plugin/hdRpr/renderDelegate.cpp]: In the blender plugin we do percent done = min ( samples/maxsamples, (num_pixels - active_pixels) / num_pixels)

Where active pixels is the one returned from adaptive sampling.

File: pxr/imaging/plugin/hdRpr/renderDelegate.cpp [hunk 2]
```diff
@@ -311,7 +311,15 @@ VtDictionary HdRprDelegate::GetRenderStats() const {
...
     int numCompletedSamples = m_rprApiSharedPtr->GetNumCompletedSamples();
     stats[HdPerfTokens->numCompletedSamples.GetString()] = numCompletedSamples;
-    stats[_tokens->percentDone.GetString()] = 100.0 * double(numCompletedSamples) / HdRprConfig::GetInstance().GetMaxSamples();
+
+    double percentDone = double(numCompletedSamples) / HdRprConfig::GetInstance().GetMaxSamples();
+    int numActivePixels = m_rprApiSharedPtr->GetNumActivePixels();
+    if (numActivePixels != -1) {
+        auto aovSize = m_rprApiSharedPtr->GetAovSize(m_rprApiSharedPtr->GetActiveAov());
+        int numPixels = aovSize[0] * aovSize[1];
+        percentDone = std::min(percentDone, double(numPixels - numActivePixels) / numPixels);
```
Comment (bsavery) [inline pxr/imaging/plugin/hdRpr/renderDelegate.cpp]: I think this want's to be a max not a min?  because we cut off renders if numActivePixels = 0

Comment (hshakula) [reply pxr/imaging/plugin/hdRpr/renderDelegate.cpp]: Yeah, you are right, I just blindly implemented your suggestion from the previous comment

Comment (hshakula) [reply pxr/imaging/plugin/hdRpr/renderDelegate.cpp]: Done

Comment (bsavery) [reply pxr/imaging/plugin/hdRpr/renderDelegate.cpp]: Well that's your fault for doing what I said.  :)

Comment (hshakula) [reply pxr/imaging/plugin/hdRpr/renderDelegate.cpp]: Your suggestion was so obvious that it passed my trust threshold

**Pull request merged**